packages/node_modules/@webex/plugin-logger/README.md

```diff
@@ -40,6 +33,38 @@ webex.logger.get(id)
 
 ```
 
+### Levels
+
+This logger plugin supports different logging levels to control what gets output to the user's console:
+
+- **silent**
+  - Nothing prints to the console.
+- **error** [DEFAULT]
+  - Error messages meant for the developer and end user to help with troubleshooting
+  - Example: `Unable to connect to websocket on port 4343`
+- **warn**
+  - Warning messages meant to help guide developers away from potential errors
+  - Example: `The usage of person email has been deprecated`
+- **log**
+  - General output of logging messages.
+  - Example: `Websocket connected on port 4343`
+- **info**
+  - More detailed logging of SDK information.
+  - Example: `Network status changed from DISCONNECTED to ONLINE` 
+- **debug**
+  - Developer specific information, helpful with debugging issues
+  - Example: `Network packet received, contents: {data}`
+- **trace**
+  - Prints the stack trace of the current call path
+
+### Level Control
+
+The developer can control what level gets printed by setting the environment variable: `WEBEX_LOG_LEVEL`.
```

Comment:
Can this also be done via `.env` file? Maybe add that alternative way too?

Reply:
I think this one is fine
Sure you could make the argument that we shouldn't assume that people don't know what a `.env` file is, but i think it's a bit unnecessary to "bloat" up the `plugin-logger` readme with talking about a `.env` file

Reply:
I added bloat in quotes because it will only be like a sentence or even less